cognite_toolkit/cdf.py

```diff
@@ -307,6 +309,32 @@ def deploy(
             verbose=ctx.obj.verbose,
         )
 
+    if "raw" in include and (directory := (Path(build_dir) / "raw")).is_dir():
```

Comment:
Didn't @doctrino pull out data_sets to be created first in another PR? Is it then necessary to split out these into the two separate? Because of the RAW dependency? @doctrino, I guess we should rather introduce a LOADER_ORDER ordered set to use so we don't have to duplicate so much code?

Reply:
Yes, I think it is ugly. Anders and I discussed it yesterday. We have to do  `auth without recourceScope` then `dataset` then `raw` then `extraction_pipelines` before `FDM`and `the rest`, and then finish `auth with resourceScope`. But it violates DRY bigtime. Hardcoding a "load order" might fix this particular case, but it wouldn't surprise me if there were cases where the order has to be different. Anyways, open to suggestions.